content/graphing/miscellaneous/functions.md

```diff
@@ -245,7 +245,7 @@ The method can be sum/min/max/count/avg and time is in seconds. You can use eith
 
 These functions are only intended for metrics submitted as rates or counters via statsd. These functions have no effect for other metric types. For more on details about how to use `.as_count()` and `.as_rate()` see [our blog post][1].
 
-Note: The only available query with `as_count()` is `sum()` (unless using a rollup summary), which is the only mathematical accurate function with such behavior.
+Note: [The only available query with `as_count()` is `sum()`](/graphing/faq/as_count_validation) (unless using a rollup summary), which is the only mathematical accurate function with such behavior.
```

Comment:
@l0k0ms -- just stumbled upon this in the docs... can this be clarified to explain that it's only referring to the monitor rollup? As it stands now, it's not clear this comment is related to monitors, making the mention of a `sum()` function very confusing.

cc @wackywendell (related to what I mentioned in the slack channel this morning)